backend/FwLite/MiniLcm/Validators/IOrderableValidator.cs

```diff
@@ -0,0 +1,12 @@
+using FluentValidation;
+using MiniLcm.Models;
+
+namespace MiniLcm.Validators;
+
+public class IOrderableValidator : AbstractValidator<IOrderable>
```

Comment:
I don't love this name as it sounds this class is actually an interface. Maybe just remove the `I` here?